### PR TITLE
Improving documentation (missing dependencies were not mentioned)

### PR DESCRIPTION
**Pull request by MayNiklas**
Dear @dr-mod, 
thank you for publishing your program!
I really do like it as an example for using a Pi Zero W with a waveshare screen.

Unfortunately, I wasn't able to get your program to work following your documentation.

Hardware: Pi Zero W
OS: 2020-12-02-raspios-buster-armhf-lite

I'm always using the lite version of the OS since I don't need the graphical environment anyway.

I do think, that it would be beneficial for your program, to work properly on such minimal installs. It seemed, like you already had those packages installed. I decided, to improve the documentation by myself. I hope you like it.

I tested everything against multiple fresh installs of RaspiOS. 

Problems that appeared:
- APT packages being needed for waveshare library to work properly were missing: libopenjp2-7 libopenjp2-7 libatlas-base-dev
- pip3 packages being needed for waveshare library to work properly were missing: image numpy
- `cd e-Paper-master/RaspberryPi_JetsonNano/python/` did not work: the repo is called e-Paper without the master

Other improvements:
- removed redundant `cd` commands: we can clone repository's by adding a relative path into the command -> saving a few commands is especially useful when being executed in a script
- we also can install packages via pip without using `cd` to go into the directory: `pip3 install ~/e-Paper/RaspberryPi_JetsonNano/python/`
- we can do the same for executing the final python program
- using pip3 install instead of pip3 install -e -> this way, the program still works in case we would remove the library's repository

You also could take a look at my commit messages, explaining what I'm doing.

I hope that my suggested changes for your documentation are wanted! In case you don't like something: tell me! I'm going to improve.

You could, if you really want to, add `sudo apt-get install -y python3-pip git` into your README. python3, git & pip aren't preinstalled on RaspiOS.

Thanks again for sharing your code!

Kind regards,
Niki from Germany

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
...
     Interfacing Options -> SPI
    ```
-2. Install eInk display drivers
+2. Installing dependencies via apt
     ```
-    git clone https://github.com/waveshare/e-Paper.git
-    cd e-Paper-master/RaspberryPi_JetsonNano/python/
-    pip3 install -e $(pwd)
+    sudo apt update
+    sudo apt install -y libtiff5 libopenjp2-7 libopenjp2-7 libatlas-base-dev
```
Comment (dr-mod) [inline README.md:18]: It looks like, these libraries are redundant.
All dependencies required for the drivers to work are described here:
https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT

Which could be boiled down to the following: 
```
sudo apt-get install python3-pip python3-pil python3-numpy
pip3 install RPi.GPIO spidev
```
The script doesn't require any additional libraries apart from listed above.

File: README.md [hunk 2]
```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
...
     ```
-    for more information refer to: https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT
-3. Download Zero BTC Screen
+3. Install eInk display drivers
+    ```
+    pip3 install image numpy RPi.GPIO spidev Pillow
```
Comment (dr-mod) [inline README.md:22]: `Image` is a bit of an overkill I'm afraid, it helped because it includes the `Pillow` library. However, it's already covered by the discontinued `python3-pil` as per the waveshares wiki.
Therefore, we can just add the following lines to save people time reading the waweshare's wiki:
```
sudo apt-get install python3-pip python3-pil python3-numpy
pip3 install RPi.GPIO spidev
```

File: README.md [hunk 3]
```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
...
+    git clone https://github.com/waveshare/e-Paper.git ~/e-Paper
+    pip3 install ~/e-Paper/RaspberryPi_JetsonNano/python/
```
Comment (dr-mod) [inline README.md:24]: Good find on the "e-Paper-master" and -e

File: README.md [hunk 4]
```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
...
+    for more information refer to: https://www.waveshare.com/wiki/2.13inch_e-Paper_HAT
+4. Download Zero BTC Screen
     ```
-4. Run it 
+    git clone https://github.com/dr-mod/zero-btc-screen.git ~/zero-btc-screen
     ```
-    cd zero-btc-screen
-    python3 main.py
+5. Run it 
+    ```Pillow
```
Comment (dr-mod) [inline README.md:32]: Mistype?

File: README.md [hunk 5]
```diff
@@ -12,24 +12,27 @@ Bitcoin stock price for your RPi Zero
...
+    python3 ~/zero-btc-screen/main.py
```
Comment (dr-mod) [inline README.md:33]: I don't think this will work as the script includes its font by a relative path. (script needs to be rewritten)

**Comment by dr-mod** [comment]
Hi Niki,
It's great that you found this script useful and went though with testing the installation process.
And of course, every bit of help or improvement is welcome, feel free to suggest things or add a pull request.
I've left a couple of comments that need to be taken care of with regards to your pull request.
Cheers,
Dmytro

File: README.md (hunk 5 above)
Comment (MayNiklas) [reply README.md:33]: This explains why I had to manually install a font!

File: README.md (hunk 4 above)
Comment (MayNiklas) [reply README.md:32]: Yes, I have no idea on how it got in there!

File: README.md (hunk 1 above)
Comment (MayNiklas) [reply README.md:18]: Sounds great!
Would still be a great improvement, since new users wouldn't need to do the research by themselves.

I realized, that I've solved the problems are more complicated way.

**Comment by MayNiklas** [comment]
Hi @dr-mod,
thank you for taking yourself some time, reviewing my pull request!
I saw your comments. They were really helpful! I'm going to implement the suggested changes in the evening. You are 100% correct!
I have chosen a complicated way of solving dependency problems.

Have a nice day,
Niki

**Comment by MayNiklas** [comment]
As discussed earlier, I've implemented the changes. I also did a test install on a fresh pi zero.

**Comment by dr-mod** [comment]
Hey Niki, I am going to merge the changes and fix that little typo. Thanks for the PR!
